domains/certificates/Shared/AggregateRepositories/AggregateRepositoryExtensions.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using CertificateEvents.Aggregates;
+using Marten;
+
+namespace AggregateRepositories;
+
+public static class AggregateRepositoryExtensions
+{
+    public static async Task Save(this IDocumentStore store, AggregateBase aggregate, CancellationToken cancellationToken = default)
+    {
+        await using var session = store.LightweightSession();
+        // Take non-persisted events, push them to the event stream, indexed by the aggregate ID
```

Comment:
If these comments are needed i suggest extracting it to a method and remove the comments

Reply:
Fixed in d5da3a142133f8e3336749957176370f8e4bd5a4